src/main/java/net/minestom/server/listener/PlayerHeldListener.java

```diff
@@ -14,26 +14,27 @@ public static void heldListener(ClientHeldItemChangePacket packet, Player player
             return;
         }
 
-        final byte slot = (byte) packet.slot();
+        final byte newSlot = (byte) packet.slot();
```

Comment:
I think we need to add some security checks here to prevent exploits or/and cast exceptions

Reply:
We already that on line 13, we ignore packets if the slot is invalid